src/base/preferences.cpp

```diff
@@ -1164,6 +1164,16 @@ void Preferences::setTrackerPort(const int port)
     setValue(u"Preferences/Advanced/trackerPort"_qs, port);
 }
 
+bool Preferences::isTrackerPortForwardingEnabled() const
+{
+    return value(u"Preferences/Advanced/trackerPortForwarding"_qs, false);
```

Comment:
Although I strongly disapprove to use such legacy naming for new settings let it be this way for consistency with other related one until they all are updated.

Reply:
>let it be this way for consistency with other related one until they all are updated.

OK. But please remind me that great idea.